src/compose/ComposeBox.js

```diff
@@ -348,9 +348,6 @@ class ComposeBox extends PureComponent<Props, State> {
     }
     completeEditMessage();
     if (this.messageInputRef.current !== null) {
-      // Should be fixed in RN v0.63 (#4245); see
-      // https://github.com/zulip/zulip-mobile/issues/4245#issuecomment-695104351.
-      // $FlowFixMe
```

Comment:
Since the `$FlowFixMe`s start getting flagged as "unused suppression
    comments", remove them.
    
    Strangely, `this.textInputRef.current` and friends are
    `any(implicit)` after this.

Hmm -- that is concerning. And indeed if I go and change these method calls to nonsense names, I get no error.

I think this may ultimately all be down to bugs in Flow's special support for React. Both `React$ElementRef`, and `React$AbstractComponent` aka `React.AbstractComponent`, which is part of the `TextInput` type, are implemented by special logic within Flow. I suspect that that logic isn't fully working as intended.

If we're not going to get type-checking on these anyway, I think I'd rather have the fixmes there, so it's obvious looking at the code that it isn't type-checked and you need to be careful about the types by hand.